gcloud/pubsub/subscription.py

```diff
@@ -259,18 +259,16 @@ def acknowledge(self, ack_ids, client=None):
                        ``client`` stored on the current subscription's topic.
         """
         client = self._require_client(client)
-        data = {'ackIds': ack_ids}
-        client.connection.api_request(
-            method='POST', path='%s:acknowledge' % (self.path,), data=data)
+        client.connection.subscription_acknowledge(self.full_name, ack_ids)
 
-    def modify_ack_deadline(self, ack_id, ack_deadline, client=None):
+    def modify_ack_deadline(self, ack_ids, ack_deadline, client=None):
```

Comment:
This change goes beyond re-mixing the API calls.

1. Why is it needed?
1. Do the system tests need updating?
1. Shouldn't it be renamed to `modify_ack_deadlines`?